pyproject.toml

```diff
@@ -65,8 +65,6 @@ dependencies = [
 ]
 dynamic = ['version', 'readme']
 
-entry-points.hypothesis = {_ = 'pydantic._hypothesis_plugin'}
```

Comment:
Unrelated to this PR but caused issues for me locally

Reply:
Maybe a better idea to comment this out rather than remove it entirely? So we at least remember to look into it? Maybe a non-issue if it will be necessary again when we work on the hypothesis plugin again..

Reply:
We’ll need to add it back to get hypothesis working, so I don’t think there’s much of a point in keeping it around